src/components/login/login.services.js

```diff
@@ -0,0 +1,22 @@
+
+// export default request => {
```

Comment:
if every service have one of these, that's a lot of duplicated code, maybe we can abstract away some of the code?

Reply:
Yeah we could have a common service